client/src/controllers/SlugController.ts

```diff
@@ -0,0 +1,28 @@
+import { Controller } from '@hotwired/stimulus';
+import { cleanForSlug } from '../utils/text';
+
+/**
+ * Adds ability for the input element value to be slugified when out of focus or blurred.
+ *
+ * @example
+ * <input
+ * type="text"
+ * name="slug"
+ * value="slug testing page"
+ * maxlength="255"
+ * required=""
+ * id="id_slug">
+ */
```

Comment:
```suggestion
 * Adds ability to slugify the value of an input element.
 *
 * @example
 * <input type="text" name="slug" id="id_slug" data-controller="w-slug" data-action="blur->w-slug#slugify" />
 */
```

This example lost the data attributes, best to double check the example is correct, also I think we should try to keep these descriptions as generic and not mention the actions used.